_data/navigation.js

```diff
@@ -162,8 +162,8 @@ module.exports = {
                 url: '/docs/advanced-tutorial/',
               },
               {
-                title: 'Developer Tutorials',
-                url: '/docs/developer-tutorials/',
+                title: 'Tutorial Reference',
```

Comment:
I'm not completely satisfied with "Tutorial Reference" as a title, but for now that's what I suggest.

Reply:
What about "Other Tutorials"?

Reply:
I'm also not certain it deserves top billing in the introduction section, vs the reference sections lower in the navigation.

Reply:
The challenge here is that we have only one Tutorials section and right now its in our introduction. Long-term we might distinguish between "Getting Started" and "Tutorials", which would resolve the question of where this content should live in the navigation.

"Other Tutorials" is fine as a title for now. It flows acceptably out of the beginner tutorials, I think.